pages/advanced-algorithms/available-algorithms/collections.mdx

```diff
@@ -29,75 +32,66 @@ sorting, and modification for efficient data handling.
 | **Edge weights**    | weighted   |
 | **Parallelism**     | sequential |
 
-## Procedures
+## Functions
 
 If a function requires elements to be comberable, `Path`, `List` and `Map` are
 uncomparable. <br />
 Only `Numeric` data types can be used in `sum()` and `avg()` functions, so only
 `Int` and `Double` data types are allowed.
 
-### `remove_all()`
+### `sort()`
```

Comment:
In the old docs, we wrote the header with function(arguments), I see that you changed it to just function(), is this something we should abide in the future?

Reply:
Yup... so please check these first couple of algorithms -> those are the ones i managed to polish... each weak i will polish 5-6 algorithms and the idea is to have them all look alike... so these are kind of the templates i'll be changing all the rest of them to

Reply:
Looks good!